### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Margen67**
See commits.

### CODE REVIEW

File: patches/5454082B - Red Dead Redemption - GOTY - Disc 1.patch
Comment (illusion0001) [inline patches/5454082B - Red Dead Redemption - GOTY - Disc 1.patch:14]: Should mention these are 2 byte float.

File: patches/584111E8 - State of Decay (TU5).patch
Comment (illusion0001) [inline patches/584111E8 - State of Decay (TU5).patch:3]: Unnecessary exclamation point

File: patches/545407DF - Rockstar Table Tennis.patch
Comment (illusion0001) [inline patches/545407DF - Rockstar Table Tennis.patch:5]: Should make certain company slightly happier

File: patches/4D530A26 - Gears of Wars Judgement.patch
Comment (illusion0001) [inline patches/4D530A26 - Gears of Wars Judgement.patch:50]: Sus whitespaces

File: patches/4D530910 - Forza Motorsport 4.patch
Comment (illusion0001) [inline patches/4D530910 - Forza Motorsport 4.patch:4]: Commented media id seem kinda useless now that we have executable hash

**Review by illusion0001 — APPROVED** [review]
Looks good. Readme is looking pretty as well, will wait for 😽 approval.

File: patches/4D530910 - Forza Motorsport 4.patch
Comment (Margen67) [reply patches/4D530910 - Forza Motorsport 4.patch:4]: Should they all be removed?

Comment (illusion0001) [reply patches/4D530910 - Forza Motorsport 4.patch:4]: Yes.

**Review by illusion0001 — APPROVED** [review]
Commit looks good

File: patches/4D530A26 - Gears of Wars Judgement.patch
Comment (Margen67) [reply patches/4D530A26 - Gears of Wars Judgement.patch:49]: Should there be a space after comment blocks?

Comment (illusion0001) [reply patches/4D530A26 - Gears of Wars Judgement.patch:49]: It depends if the prior line of comment is behind a line of code then yea.
```
My_code ; my comment
        ; my comment 2
```

Comment (illusion0001) [reply patches/4D530A26 - Gears of Wars Judgement.patch:49]: This change is good.

**Comment by Margen67** [comment]
Should this be moved to the GitHub repo? https://cdn.discordapp.com/attachments/747164286056661153/764464248176115712/unknown.png

**Comment by illusion0001** [comment]
Sure if you can take a higher resolution image of that and add it to `/docs/images` or something

**Comment by Margen67** [comment]
> Sure if you can take a higher resolution image of that and add it to `/docs/images` or something

[](https://github.com/Margen67/game-patches/blob/memes/images/patches.png)

**Comment by illusion0001** [comment]
> [](https://github.com/Margen67/game-patches/blob/memes/images/patches.png)

Looks good

**Comment by bomabomaboma** [comment]
For the UE games "Disable Lens Flare" should probably be changed to "Disable Lens Flares" (proper verbiage) and I think the desc of _"Fixes unoccluded bright lights on-screen."_ might be the best uniform descriptor for the layman who won't understand _"Solves most occlusion related issues"_.

Thanks for all the awesome copy editing.

**Pull request merged**
